best-practices.md

```diff
@@ -297,6 +298,42 @@ is strongly recommended that Catalogs don't contain differently versioned Items
 consistent (Sub-)Catalogs containing either all or no data. Collections that are referenced from Items should always use the same
 STAC version. Otherwise some behaviour of functionality may be unpredictable (e.g. merging common fields into Items or reading summaries).
 
+## Using Summaries in Collections
+
+One of the strongest recommendations for STAC is to always provide [summaries](collection-spec/collection-spec.md#summaries) in
+your collections. The core team decided to not require them, in case there are future situations where providing a summary
+is too difficult. The idea behind them is not to exhaustively summarize every single field in the collection, but to provide
+a bit of a 'curated' view. 
+
+Any field that is a range of data (like numbers or dates) is a great candidate to summarize, to give people a sense what values
+the data might be. For example in overhead imagery, a 
+[`view:off_nadir`](extensions/view/README.md#item-properties-and-item-asset-fields) with a range of 0 to 3 would tell people this 
+imagery is all pretty much straight down, while a value of 15 to 40 would tell them that it's oblique imagery, or 0 to 60 that it's 
+a collection with lots of different look angles. 
+
+Fields that have only one or a handful of values are also great to summarize. Collections with a single satellite may
+use a single [`gsd`](item-spec/common-metadata.md#instrument) field in the summary, and it's quite useful for users to know
+that all data is going to be the same resolution. Similarly it's useful to know the names of all the 
+[`platform` values](item-spec/common-metadata.md#instrument) that are used in the collection. 
+
+It is less useful to summarize fields that have numerous different discrete values that can't easily be represented
+in a range. These will mostly be string values, when there aren't just a handful of options. For example if you had a 
+'location' field that gave 3 levels of administrative region (like 'San Francisco, California, United States') to help people
+understand more intuitively where a shot was taken. If your collection has millions of items, or even hundreds, you don't want
+to include all those values as a summary. 
+
+We do highly recommend including an [`eo:bands`](extensions/eo/README.md#eobands) summary if your Items implement `eo:bands`, 
```

Comment:
Did we somewhere explain that some arrays can be merged and some shouldn't be merged? I thought we had, but I can't find it right now.
Like eo:bands makes sense to be merged, but on the other hand proj:shape shouldn't be merged at all in the summary.

Reply:
Yeah, that does sound familiar. I'll try to find it. Though do we have any guidance as to what can be merged and what can't?

Reply:
Didn't find it, but made an attempt to cover it. Let me know if I got anything wrong in it.

Reply:
Looks good/better. We could maybe link to the stac-fields json file and say that there are indicatiors for the array merge included?

Reply:
Or do you think it would make sense to actually give the array merge details for summaries directly in the spec for each of the array-fields? Could be useful indeed.

Reply:
I'm more inclined to link to stac-fields json. I'm not sure that we want to lock-in exactly how people should do array merges. To me the core spec should be flexible, with tooling/best practices explaining practically how most people do things. This falls into that camp for me.

Reply:
I've read it again and indeed there's nothing to do here, I had understood the "judgement call" differently before.